pages/landing/patcher/contact/_form.html

```diff
@@ -16,9 +16,47 @@
   <input type="hidden" name="hello_gruntwork" value=""/>
   <input type="hidden" name="_landing-page" value="{{ site.url }}{{ page.url }}" />
   <input type="text" name="_gotcha" style="display:none" />
-  {% include_relative _input-text.html id="contact-name" label="Name" required=true placeholder="Jon Doe" %}
-  {% include_relative _input-text.html id="contact-email" label="Email" required=true placeholder="jon@acme.com" type="email" %}
+  {% include_relative _input-text.html id="contact-name" label="Name" required=true placeholder="Jane Doe" %}
+  {% include_relative _input-text.html id="contact-email" label="Email" required=true placeholder="jane@acme.com" type="email" %}
   {% include_relative _input-text.html id="contact-company" label="Company" required=true placeholder="Acme, Inc." %}
+  <div class="form-group">
```

Comment:
Somewhere in here, we'll want to capture which subscription / pricing option they picked... Once we have a pricing page of course. Perhaps file a GitHub issue to capture that?

Reply:
Ah, good.

Reply:
I created https://github.com/gruntwork-io/gruntwork-io.github.io/issues/554 and set it to `patcher-v2`